### ISSUE TITLE
Request by TLSv1

### ISSUE THREAD

**Issue by SeleznevAM** [kittinunf/fuel#692]
Hi all.
I have api client on Fuel.
Sometimes, if I have a bad network connection I observe what my request going by TLSv1.
it happens when I use my application on Android 8 or 9.
What must I get to you to resolve my trouble?
I have Charles dump and Wireshark dump where you can see it.
Advance thanks.

**Comment by kittinunf** [comment]
Can you post your snippet and explain what went wrong? Sorry I couldn't grasp your question 🙏

**Comment by iNoles** [comment]
You wanted Fuel to force TLSv1 in certain url addresses?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by SeleznevAM** [comment]
> Can you post your snippet and explain what went wrong? Sorry I couldn't grasp your question 🙏

Yes of course, but only tomorrow.

**Comment by SeleznevAM** [comment]
> You wanted Fuel to force TLSv1 in certain url addresses?
> 
> Sent with [GitHawk](http://githawk.com)

No. I want to exclude TLS1 from client-server communicating. Now, how I think, if I have a bad internet connection, during ssl handshake session, my application sends packages by TLS1 protocol. And I want to exclude these situations
